Automattic/wp-calypso
# Site Profiler: switches migration banner to footnote.

## Proposed Changes

* Replaces the small migration banner with a footnote explaining CRUX results as per new designs

![CleanShot 2024-05-31 at 14 41 49@2x](https://github.com/Automattic/wp-calypso/assets/12430020/e5336c88-9b5b-47c3-bb33-8d6ff98a92f7)

## Testing Instructions



* Go to `/site-profiler` 
* Inspect the banner on the bottom
* Make sure it still displays in the results page


https://github.com/Automattic/wp-calypso/assets/12430020/f79641d2-ca8f-40d0-841d-69d26e05b23d



## Pre-merge Checklist
